### PR TITLE
fix: Standardize width of footer

### LINKED ISSUE #5511: Public Event Pages: Standardize width of sidebar and make it same as the footer
On the public event pages the sidebar spacing is different to the footer left and right spacing. The frontpage has the same left and right spacing as the footer.

Expected is: that on the public event pages the left and right space needs to be same as on the footer. The central content column needs to have the same width as the central area of the footer.

Screenshot from 2020-11-06 17-11-29

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #5511 

#### Short description of what this resolves:
On the public event pages the sidebar spacing is different to the footer left and right spacing. This PR matched the left and right spacing of footer with sidebar and social links widget of public event page.

The footer is having a grid of 4 column which is wrapped in a center aligned container.
I have removed the container and created a grid with 3 columns. the 2 side column are empty(to match space with side bar of public event page and right side twitter widget). the center column is a nested grid which contain all footer data(grid of footer and all four columns.).
now footer is matched with the public event page.

#### Changes proposed in this pull request:

- matched the left and right spacing of footer with public event page side bar.
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

__screenshots__

__before__

__after__

### CODE REVIEW

**Comment by sachinchauhan2889** [comment]
The footer is having a grid of 4 column which is wrapped in a center aligned container.
I have removed the container and created a grid with 3 columns. the 2 side column are empty(to match space with side bar of public event page and right side twitter widget).  the center column is a nested grid which contain all footer data(grid of footer and all four columns.).
now footer is matched with the public event page.

File: app/templates/components/footer-main.hbs [hunk 1]
```diff
@@ -1,127 +1,130 @@
...
+              <div class="item" data-value="{{key}}">{{value}}</div>
+            {{/each-in}}
+          </div>
+        </UiDropdown>
+      </div>
     </div>
   </div>
-</div>
+  <div class="three wide column"></div>
+</div>
```
Comment (Raunaksingh100) [inline app/templates/components/footer-main.hbs:130]: add a new line

**Review by Raunaksingh100 — CHANGES_REQUESTED** [review]
add a new line

**Review by mariobehling — APPROVED** [review]
Works for me.

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Why is there a need of such drastic structural changes? Please make minimum changes necessary

**Comment by sachinchauhan2889** [comment]
@iamareebjamal sir, please have a look at changes done by me. i have done minimum changes needed. I have removed the outer container and replace it with a Semantic UI Grid. 
In github it is showing i have changed 117 lines but actually i have changed 2 line and add 3 lines. i have screenshot of VS Code diaplaying changes done by me. please have a look at this.

5511 chat

**Comment by sachinchauhan2889** [comment]
@iamareebjamal sir, have a look at this. There is no such drastic change in structure. Only outer centered aligner container is replaced with semantic ui grid and for matching left side bar and right twitter widget, 2 empty columns are added.
I think it is good approach for this problem.
Sir if you told to change, i will think for another solution.

File: app/templates/components/footer-main.hbs (hunk 1 above)
Comment (iamareebjamal) [inline app/templates/components/footer-main.hbs:130]:
```suggestion
</div>

```

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**